build/configs/rtl8721csm/tc/defconfig

```diff
@@ -175,8 +192,9 @@ CONFIG_ARCH_USE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="4096,"
+CONFIG_FLASH_PART_SIZE="128,"
```

Comment:
Is it okay to set the smartfs part only?

Reply:
Because we use different flash memories in the development board, so the flash address for smartfs and also the partition size needs to be determined by the user.